kapitan/inputs/jinja2_filters.py

```diff
@@ -95,6 +96,10 @@ def to_yaml(obj):
     return yaml.safe_dump(obj, default_flow_style=False)
 
 
+def to_toml(obj):
```

Comment:
You'll need to add this to the list in function load_jinja2_filters(env) for this to work. Also would be good to update docs to include this too and add a small test for it. :)

Reply:
Will do! I am actually working on the tests for the env reftype right now. I will also update all the docs so that folks aren't left in the dark.